ambari-common/src/main/python/resource_management/libraries/functions/get_not_managed_resources.py

```diff
@@ -34,8 +35,8 @@ def get_not_managed_resources():
   """
   config = Script.get_config()
   not_managed_hdfs_path_list = json.loads(config['hostLevelParams']['not_managed_hdfs_path_list'])[:]
-  if 'managed_hdfs_resource_property_names' in config['configurations']['cluster-env']:
-    managed_hdfs_resource_property_names = config['configurations']['cluster-env']['managed_hdfs_resource_property_names']
+  if get_cluster_setting('managed_hdfs_resource_property_names') not None:
```

Comment:
I think `if ... not None:` is invalid syntax.  It should be `if ... is not None:` instead.  Sorry for not noticing it earlier.

```
>>> def x():
...     return 'x'
...
>>> if x() not None:
  File "<stdin>", line 1
    if x() not None:
                  ^
SyntaxError: invalid syntax
```